harvard-imperial-college-london.csl

```diff
@@ -8,15 +8,18 @@
     <author>
       <name>Lawrence Greenfield</name>
     </author>
+	<contributor>
+      <name>S. Mackenzie</name>
+    </contributor>
     <category citation-format="author-date"/>
     <category field="medicine"/>
-    <summary>Harvard author-date style edited for Imperial College London</summary>
-    <updated>2017-04-27T21:09:20+00:00</updated>
+    <summary>Harvard style from Imperial College London Library  - 2021</summary>
+    <updated>2021-09-28T14:37:15+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="editor">
     <names variable="editor" delimiter=", ">
-      <name and="symbol" delimiter=", "/>
+      <name and="symbol" initialize="false"/>
```

Comment:
why initialize="false" here? I'm seeing initials used for all creators, so if anything shouldn't it be `initialize-with="."`?

Reply:
done

Reply:
agreed- changed